[2025-08-28T02:20:08.091+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-28T02:20:08.132+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_generate_orders.generate_new_data manual__2025-08-28T02:20:05.460735+00:00 [queued]>
[2025-08-28T02:20:08.147+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_generate_orders.generate_new_data manual__2025-08-28T02:20:05.460735+00:00 [queued]>
[2025-08-28T02:20:08.147+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-28T02:20:08.174+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_new_data> on 2025-08-28 02:20:05.460735+00:00
[2025-08-28T02:20:08.183+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=1781) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-28T02:20:08.185+0000] {standard_task_runner.py:64} INFO - Started process 1783 to run task
[2025-08-28T02:20:08.186+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'daily_generate_orders', 'generate_new_data', 'manual__2025-08-28T02:20:05.460735+00:00', '--job-id', '749', '--raw', '--subdir', 'DAGS_FOLDER/generate_fake_shop_csv.py', '--cfg-path', '/tmp/tmpd3d5zdzq']
[2025-08-28T02:20:08.190+0000] {standard_task_runner.py:91} INFO - Job 749: Subtask generate_new_data
[2025-08-28T02:20:08.211+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-08-28T02:20:08.248+0000] {task_command.py:426} INFO - Running <TaskInstance: daily_generate_orders.generate_new_data manual__2025-08-28T02:20:05.460735+00:00 [running]> on host b5ea13053d47
[2025-08-28T02:20:08.367+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='daily_generate_orders' AIRFLOW_CTX_TASK_ID='generate_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-28T02:20:05.460735+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-28T02:20:05.460735+00:00'
[2025-08-28T02:20:08.368+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-28T02:20:08.968+0000] {logging_mixin.py:188} INFO - ✅ Добавлено 10 новых заказов с корректными значениями и переставленными order_amount
[2025-08-28T02:20:08.970+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-28T02:20:08.971+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-28T02:20:08.979+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=daily_generate_orders, task_id=generate_new_data, run_id=manual__2025-08-28T02:20:05.460735+00:00, execution_date=20250828T022005, start_date=20250828T022008, end_date=20250828T022008
[2025-08-28T02:20:09.004+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-28T02:20:09.022+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-28T02:20:09.027+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
